src/Common/src/Interop/Linux/procfs/Interop.ProcFsStat.cs

```diff
@@ -78,6 +80,17 @@ internal struct ParsedStat
             //internal long cguest_time;
         }
 
+        internal struct ParsedStatus
+        {
+            internal int pid;
```

Comment:
unless this is matching C-struct then please rename this to `Pid`

Reply:
@krwq Done.